src/Phinx/Migration/Manager.php

```diff
@@ -675,7 +675,7 @@ public function getMigrations()
                     }
 
                     // instantiate it
-                    $migration = new $class($version, $this->getInput(), $this->getOutput());
+                    $migration = new $class($this, $version, $this->getInput(), $this->getOutput());
```

Comment:
OOI, as `$this` is being supplied, why bother supplying the `input` and `output`?